testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/microprofile/config/MicroProfileConfigFilterTest.java

```diff
@@ -134,28 +129,12 @@ public void testContextInject() throws Exception {
 
    /**
     * @tpTestDetails Verify former context parameter "resteasy.add.charset" is overridden by system property.
-    * @tpSince RESTEasy 3.10.0
+    * @tpSince RESTEasy 4.0.0
     */
    @Test
    public void testActualContextParameter() throws Exception {
       Response response = client.target(generateURL("/actual")).request().get();
       Assert.assertEquals(200, response.getStatus());
       Assert.assertEquals("text/plain", response.getHeaderString("Content-Type"));
    }
-
-   /**
-    * @tpTestDetails Verify that ConfigSource names are retrieved properly.
-    * @tpSince RESTEasy 3.12.0
-    */
-   @Test
-   public void testConfigSourceNames() throws Exception {
```

Comment:
Does this not work any more?

Reply:
The tests in testsuite/integration-tests were updated to those that reside in master.  That test does not reside in master.